jest.config.ts

```diff
@@ -0,0 +1,61 @@
+import type { Config } from "jest";
+
+const config: Config = {
+  preset: "ts-jest",
+  verbose: true,
+  projects: [
+    {
+      displayName: "@calcom/web",
+      roots: ["<rootDir>/apps/web"],
+      moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+      modulePathIgnorePatterns: [
+        //
+        "<rootDir>/apps/web/test/__fixtures__",
+        "<rootDir>/apps/web/node_modules",
+        "<rootDir>/apps/web/dist",
+      ],
+      clearMocks: true,
+      setupFilesAfterEnv: ["<rootDir>/tests/config/singleton.ts"],
+      setupFiles: ["<rootDir>/apps/web/test/jest-setup.js"],
+      testMatch: ["**/test/lib/**/*.(spec|test).(ts|tsx|js)", "**/__tests__/**/*.(spec|test).(ts|tsx|js)"],
```

Comment:
I would go only leaving option two, and renaming actual test/lib to _tests_.
```suggestion
      testMatch: ["**/__tests__/**/*.(spec|test).(ts|tsx|js)"],
```

Reply:
Good for a followup